### PR TITLE
i18n(de): update `editor-setup.mdx`

### PR DESCRIPTION
**Pull request by DerTimonius**
#### Description (required)

This PR will update the german translation for the `editor-setup` documentation to match the english version. I also made a small change to the english version, because Fleet has been released some time ago.

### CODE REVIEW

File: src/content/docs/en/editor-setup.mdx [hunk 1]
```diff
@@ -28,7 +28,7 @@ import ReadMore from '~/components/ReadMore.astro';
 
 Initial support for Astro landed in WebStorm 2023.1. You can install the official plugin through [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/20959-astro) or by searching for "Astro" in the IDE's Plugins tab. This plugin includes features like syntax highlighting, code completion, and formatting, and plans to add even more advanced features in the future. It is also available to all other [JetBrains IDEs with JavaScript support](https://www.jetbrains.com/products/#lang=js&type=ide).
 
-JetBrains' upcoming [Fleet IDE](https://www.jetbrains.com/fleet/) includes language server support and will be able to use the official Astro tooling.
+JetBrains' new [Fleet IDE](https://www.jetbrains.com/fleet/) includes language server support and will be able to use the official Astro tooling.
```
Comment (sarah11918) [inline src/content/docs/en/editor-setup.mdx:31]:
```suggestion
JetBrains' [Fleet IDE](https://www.jetbrains.com/fleet/) includes language server support and uses the official Astro tooling.
```
Just like "upcoming" gets out of date, so does "new"! Better to not have any language like this for future-proofing! You can update the German accordingly to match this version, too.

Comment (DerTimonius) [reply src/content/docs/en/editor-setup.mdx:31]: Makes sense, will change this!

**Review by Maxframe — APPROVED** [review]
At a first glance, I couldn't find any typos or expression errors. Great work!

**Comment by Yan-Thomas** [comment]
Thanks @DerTimonius, nice work! Can I ask for you to move the change to the English version to its own PR? This way its easier for others translators to see the change to the source file while updating their versions, after doing so I'll merge both PRs. 🙌

**Comment by DerTimonius** [comment]
> Thanks @DerTimonius, nice work! Can I ask for you to move the change to the English version to its own PR? This way its easier for others translators to see the change to the source file while updating their versions, after doing so I'll merge both PRs. 🙌

Sure thing, removed the change here and opened a new PR: #6532

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work 🙌

**Pull request merged**